xtp/src/libxtp/activedensitymatrix.cc

```diff
@@ -0,0 +1,80 @@
+/*
+ *            Copyright 2009-2022 The VOTCA Development Team
+ *                       (http://www.votca.org)
+ *
+ *      Licensed under the Apache License, Version 2.0 (the "License")
+ *
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *              http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * References- (1) A fast intrinsic localization procedure applicable for ab
```

Comment:
Move he references out of the copyright header - unless that’s what we have done earlier?

Reply:
It was advised in one of the earlier comments to do this.